modules/jwt_auth_refresh/src/JwtRefreshTokensInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Drupal\jwt_auth_refresh;
+
+use Drupal\Core\Session\AccountInterface;
+
+interface JwtRefreshTokensInterface {
```

Comment:
I think you mean `ContentEntityBase`? I believe that brings along a bunch of extra cruft we don't need, so I just went with this.